.codeclimate.yml

```diff
@@ -0,0 +1,32 @@
+version: "2"         # required to adjust maintainability checks
+
+checks:
+  argument-count:
+    enabled: false
+  file-lines:
+    enabled: false
+  method-count:
+    enabled: false
+  method-lines:
+    enabled: false
+  method-complexity:
+    config:
+      # Set it to the same value as Sonar
```

Comment:
We can probably get rid of this comment since we aren't replacing Sonar in this repo.